addon/utils/ce/paste-handler-func.ts

```diff
@@ -0,0 +1,12 @@
+import { cleanDocx } from '@prezly/docx-cleaner';
+
+export default function pasteHandler(
+  rtfPaste: string,
+  htmlPaste: string
+): string {
+  if (!rtfPaste && !htmlPaste) return 'No content found!';
```

Comment:
I should add a linting rule for this, but the convention in most of our projects is to always use braces for conditionals, even if they can be oneliners
it's just a style choice really, we feel like its easier to make mistakes with braceless conditionals